cmd/spire-server/cli/run/run.go

```diff
@@ -74,6 +75,7 @@ type serverConfig struct {
 	RegistrationUDSPath string             `hcl:"registration_uds_path"`
 	DefaultSVIDTTL      string             `hcl:"default_svid_ttl"`
 	TrustDomain         string             `hcl:"trust_domain"`
+	AttestLimit         int                `hcl:"attest_limit"`
```

Comment:
I wonder if this should be in a "ratelimit" group or something... the attestation limit might not be the only thing that may need to be adjusted running this at scale through an IP-based load balancer. Its possible that we'll have to add configurables for the other limits.

Reply:
That makes sense to me, I'll make a `ratelimit` group and put an `attestation` configurable inside.